### PR TITLE
[FDP-623] Geen dimValue teruggeven in GetStatusResponse als relais uit staat (voor zowel SSLD als LMD)

### PR DESCRIPTION
**Pull request by jasperkamerling**

### CODE REVIEW

File: integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@PublicLighting @Platform @PublicLightingSetLightSchedule
+@PublicLighting @Platform @PublicLightingSetLightSchedule @Jasper
```
Comment (JelleHoffman) [inline integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature]: Development code?

**Review by JelleHoffman — DISMISSED** [review]

File: integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature (hunk 1 above)
Comment (jasperkamerling) [reply integration-tests/cucumber-tests-platform-publiclighting/src/test/resources/features/publiclighting/osgp-adapter-ws-publiclighting/ScheduleManagement/SetLightSchedule.feature]: oops, is nu weer weg

File: osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/application/mapping/converters/LightValueConverter.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2021 Alliander N.V.
```
Comment (smvdheijden) [inline osgp/platform/osgp-adapter-ws-publiclighting/src/main/java/org/opensmartgridplatform/adapter/ws/publiclighting/application/mapping/converters/LightValueConverter.java]:
```suggestion
 * Copyright 2022 Alliander N.V.
```

File: osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/application/mapping/LightValueToOslpLightValueMapper.java [hunk 3]
```diff
@@ -28,9 +28,9 @@ public LightValueDto convertFrom(
       index = source.getIndex().byteAt(0);
     }
 
-    int dimValue = 0;
+    Integer dimValue = null;
     if (source.hasDimValue()) {
```
Comment (smvdheijden) [inline osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/application/mapping/LightValueToOslpLightValueMapper.java]:
```suggestion
    if (source.hasDimValue() && source.hasOn()) {
```

**Review by smvdheijden — CHANGES_REQUESTED** [review]
- Looks like this pull request contains several formatting changes in the feature files. Check if you could keep numbers right-aligned. In Eclipse this could be set in the preferences, not sure about IntelliJ.
- You've added some additional examples to the SetLightSchedule feature, but shouldn't there be any additional examples in the GetLightStatus feature as well?

**Comment by jasperkamerling** [comment]
Intellij doesnt have a different alignment for numbers. But the eclipse allignement also doesn't really work because most of the numbers here are actually lists of numbers so those would need to go on the left.
I'll change them back for now but maybe we can look into it later and fix this in Eclipse.

File: osgp/shared/pom.xml [hunk 4]
```diff
@@ -134,6 +134,7 @@
       <modules>
         <module>osgp-kafka-logger</module>
         <module>osgp-ws-distributionautomation</module>
+        <module>osgp-ws-secret-management</module>
```
Comment (smvdheijden) [inline osgp/shared/pom.xml]: osgp-ws-secret-management should not be needed here.

**Review by smvdheijden — CHANGES_REQUESTED** [review]

**Review by smvdheijden — APPROVED** [review]

**Review by smvdheijden — APPROVED** [review]

**Pull request merged**